README.md

```diff
@@ -33,19 +33,23 @@ html(lang='en')
 Get it on [npm](https://www.npmjs.com/package/html2pug):
 
 ```bash
-npm i -g html2pug
+npm install -g html2pug
 ```
 
 ## Usage
 
 ### CLI
-Accept input from a file and write to stdout:
+Accept input from a file or stdin and write to stdout:
 
 ```bash
+# choose a file
 html2pug < example.html
```

Comment:
Maybe have `npx html2pug < example.html` instead of global install ?

Reply:
@pvalentim That's certainly possible if this package was installed locally for programmatic use, but this package kinda makes more sense as a global CLI tool. Could you clarify?